makefile

```diff
@@ -13,8 +13,15 @@ running\:%:
 		-f ./running/Dockerfile ./running/
 		--progress=plain
 
+zip\:%:
+	node "twig.js" Zip_Dockerfile.twig "{\"imageName\": \"$@\"}" > Dockerfile
+	docker build --no-cache --progress=plain \
+		-t "ghcr.io/shopware5/docker-images-testing/$@" \
+		-f ./Dockerfile ./
+		--progress=plain
+
 install\:%:
-	node "twig.js" Dockerfile.twig "{\"imageName\": \"$@\"}" > Dockerfile
+	node "twig.js" Instsall_Dockerfile.twig "{\"imageName\": \"$@\"}" > Dockerfile
```

Comment:
```suggestion
	node "twig.js" Install_Dockerfile.twig "{\"imageName\": \"$@\"}" > Dockerfile
```

Reply:
This hangs together with the wrong file name